src/ImageSharp/Processing/Processors/Convolution/MedianRowOperation{TPixel}.cs

```diff
@@ -75,7 +75,7 @@ public void Invoke(int y, Span<Vector4> span)
         // First convert the required source rows to Vector4.
         for (int i = 0; i < this.kernelSize; i++)
         {
-            int currentYIndex = Unsafe.Add(ref sampleRowBase, i);
+            int currentYIndex = Unsafe.Add(ref sampleRowBase, (uint)i);
```

Comment:
Just below `i` is needed for slicing as `int`. 
It would be more consistent, but also two casts instead of one in the loop body.

I have no strong opinion what's better here. Consistency or less casts?